theme/static/css/hydroshare_core.css

```diff
@@ -3404,6 +3404,7 @@ div.btn-update-picture:hover {
 
 #user-profile-tabs {
     margin-top: 20px;
+	margin-bottom: 60px;
```

Comment:
@mauriel133 appears to have some tab vs space misalignment in style definition above. Just something to check for to make sure they are aligned properly for readability.